miner/algo_common.go

```diff
@@ -22,10 +22,33 @@ const (
 	popTx   = 2
 )
 
+// defaultProfitPercentMinimum is to ensure committed transactions, bundles, sbundles don't fall below this threshold
+// when profit is enforced
+const defaultProfitPercentMinimum = 70
+
+var (
+	defaultProfitThreshold  = big.NewInt(defaultProfitPercentMinimum)
+	defaultValidationConfig = validationConfig{
+		EnforceProfit:          false,
+		ExpectedProfit:         common.Big0,
+		ProfitThresholdPercent: defaultProfitThreshold,
+	}
+)
+
 var emptyCodeHash = common.HexToHash("c5d2460186f7233c927e7db2dcc703c0e500b653ca82273b7bfad8045d85a470")
 
 var errInterrupt = errors.New("miner worker interrupted")
 
+type validationConfig struct {
```

Comment:
I think the name is a bit misleading, maybe algo config? apply config?

Reply:
updated to `algorithmConfig`